tests/byzantium/precompiles/test_05_modexp.py

```diff
@@ -0,0 +1,175 @@
+    """
+    Generates input for the MODEXP precompile, with the inputs `b` (base), `e` (exponent),
+    `m` (modulus) and optionally extra bytes to append at the end of the call input
+    """
+    return (
+        "0x"
+        + f"{int(len(b)/2):x}".zfill(64)
+        + f"{int(len(e)/2):x}".zfill(64)
+        + f"{int(len(m)/2):x}".zfill(64)
+        + b
+        + e
+        + m
+        + extra
+    )
+
+
+@pytest.mark.valid_from("Byzantium")
+@pytest.mark.parametrize(
+    ["input", "output"],
+    [
+        # format: ([b, e, m, extraData?], [callSuccess, returnedData])
+        # Here, `b`, `e` and `m` are the inputs to the ModExp precompile
+        # The output callSuccess is either 0 (fail of call (out-of-gas)) or 1 (call succeeded)
+        # The output returnData is the expected output of the call
+        # The optional extraData is extra padded data at the end of the calldata to the precompile
+        (["", "", "02"], ["0x01", "0x01"]),
+        (["", "", "0002"], ["0x01", "0x0001"]),
+        (["00", "00", "02"], ["0x01", "0x01"]),
+        (["", "01", "02"], ["0x01", "0x00"]),
+        (["01", "01", "02"], ["0x01", "0x01"]),
+        (["02", "01", "03"], ["0x01", "0x02"]),
+        (["02", "02", "05"], ["0x01", "0x04"]),
+        (["", "", ""], ["0x01", "0x"]),
+        (["", "", "00"], ["0x01", "0x00"]),
+        (["", "", "01"], ["0x01", "0x00"]),
+        (["", "", "0001"], ["0x01", "0x0000"]),
+        # Test cases from EIP 198 (Note: the cases where the call goes out-of-gas and the
+        # final test case are not yet tested)
+        pytest.param(
+            [
+                "03",
+                "fffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffc2e",
+                "fffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffc2f",
+            ],
+            ["0x01", "0000000000000000000000000000000000000000000000000000000000000001"],
+            id="EIP-198-case1",
+        ),
+        pytest.param(
+            [
+                "",
+                "fffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffc2e",
+                "fffffffffffffffffffffffffffffffffffffffffffffffffffffffefffffc2f",
+            ],
+            ["0x01", "0000000000000000000000000000000000000000000000000000000000000000"],
+            id="EIP-198-case2",
+        ),
+        pytest.param(
+            [
+                # Note: the only case which goes out-of-gas, and this is also raw input
+                # which is thus not fed into create_modexp_tx-_data
+                """0000000000000000000000000000000000000000000000000000000000000000"""
+                + """0000000000000000000000000000000000000000000000000000000000000020"""
+                + """ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"""
+                + """fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe"""
+                + """fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffd"""
```

Comment:
This line and the one above fail to pass the spell check... should I add this to the whitelist? (Why are the other hex strings not failing the whitelist though? )